active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,336 @@
+	        NumberOfPages="10"
+                FirstButtonVisibility="None"
+                PreviousButtonVisibility="AlwaysVisible"
+                NextButtonVisibility="AlwaysVisible"
+                LastButtonVisibility="None"
+	        EllipsisShowFirstAndLast="True">
+	</controls:PagerControl>
+</Grid>
+```
+
+### Customize edge buttons
+
+You can customize the visibility of each of the edge buttons (the first, previous, next, and last) by setting their visibility properties. In this example, the first and previous buttons will always be visible while the next button will be hidden when the last page is selected. The last button will be hidden. By default, all buttons will be set to AlwaysVisible.
+
+XAML
+```XAML
+<Grid>
+        <!-- ListView Layout -->
+        <controls:PagerControl x:Name="MainPagerControl"
+                DisplayMode="ComboBox"
+                NumberOfPages="10"
+                FirstButtonVisibility="AlwaysVisible"
+                PreviousButtonVisibility="AlwaysVisible"
+                NextButtonVisibility="HiddenOnEdge"
+                LastButtonVisibility="None"
+                Prefix="Page"
+                Suffix="of 10"/>
+        </controls:PagerControl>
+</Grid>
+```
+
+### Handling a page change event
+
+When a user interacts with the pager control and selects an edge button or page number, the page changed event will be fired. This event will provide the index the user is coming from (previousPage) and the index the user is navigating to (currentPage). Below is a code example of how to handle this event. 
+
+XAML
+```XAML
+<Grid>
+        <!-- DataGrid Layout -->
+        <controls:PagerControl x:Name="MainPagerControl"
+                DisplayMode="Auto"
+                NumberOfPages="10"
+                FirstButtonVisibility="None"
+                PreviousButtonVisibility="AlwaysVisible"
+                NextButtonVisibility="AlwaysVisible"
+                LastButtonVisibility="None"
+                Prefix="Page"
+                Suffix="of 10"/>
+        </controls:PagerControl>
+</Grid>
+```
+C#
+(insert code sample)
+```C#
+```
+
+## API Notes
+| Name | Description| Default | 
+|:---:|:---|:---|
+| PagerDisplayMode | Enum that contains 4 values (Auto, ComboBox, NumberBox, ButtonPanel) to represent the look of the pager control. When Auto is selected, the display mode will be ComboBox if the NumberOfPages property is less than 11 otherwise it will be NumberBox. | Auto |
+| PagerButtonVisibilityBehavior | Enum that contains 4 values (Auto, AlwaysVisible, HiddenOnEdge, None) that allows the app developer to hide or show the four edge buttons. HiddenOnEdge will remove the appropriate buttons if the selected page is the last or first page. When the last page is selected, the next and last buttons will be disabled and same when the first page is selected for the first and previous page. When Auto is selected, the visibility mode will be AlwaysVisible. | Auto |
```

Comment:
nit: Is none the same as AlwaysHidden ?

Reply:
Yes.

Reply:
@ranjeshj  are you suggesting this should be hidden instead of none? I think @MikeHillberg agreed that none is fine.

Reply:
It wasn't very obvious to me but I'll let the API review crew decide the naming.

Reply:
All the precedent, including in this enum, is for "Hidden". (I think just Hidden, not AlwaysHidden.)